### ISSUE TITLE
can't switch locale

### ISSUE THREAD

**Issue by emdete** [fsinfuhh/Bitpoll#40]
i would like to use a german locale but it stays english. i found `LANGUAGE_CODE = 'de-de'` in `bitpoll/settings.py` by default (while `#LANGUAGE_CODE = 'en-us'` is in `bitpoll/settings_local.sample.py` - wasnt that meant to doc the default?) but however i change that setting i see english.

i wonder why that is "en-us" while the name of the locale is "en_US", but even if i set it to en_US or de_DE it doesnt work for me. what did i do wrong? i blindly followed the instructions in the README...

**Comment by hprid** [comment]
Did you compile the gettext files using `./manage.py compilemessages`? If yes, there should be *.mo files next to the *.po files. Loading new *.mo files requires a restart of the application server.

**Comment by emdete** [comment]
yes, i did a `compilemessages` and restarted the server afterwords.

**Comment by emdete** [comment]
It shows as well " This poll has a different timezone than you." while both should be the same. How can i debug the issue to fix it?
